wiki/People/Global_Moderation_Team/vi.md

```diff
@@ -0,0 +1,82 @@
+3. Đừng trả lời câu hỏi của người khác nếu không biết chính xác câu trả lời, nhất là khi ở trên diễn đàn. Gợi ý, đề xuất là tốt, nhưng đưa ra thông tin sai lệch thì sẽ chẳng giúp ích cho ai cả. Nếu không biết, hãy để người khác giải quyết và học hỏi từ họ.
+4. Đừng làm quá sức mình. Đội ngũ chú tâm đến sự đóng góp đều đặn thường xuyên, nên ôm đồm nhiều việc trong thời gian ngắn không phải là cách tốt. Điều này thường dẫn tới việc bị kiệt sức, gây ra việc mất đi sự kiên nhẫn và sự tận tâm với công việc của mình. Bình tĩnh thôi!
+
+## Lợi ích
+
+Kiểm duyệt viên sẽ được hưởng những lợi ích sau:
+
+- **[osu!supporter](/wiki/osu!supporter)**: Một thẻ osu!supporter có hiệu lực trong suốt khoảng thời gian làm thành viên của đội ngũ.
+- **Huy hiệu nhiệm kỳ**: Một [huy hiệu hồ sơ](/wiki/Community/Profile_badge) thể hiện số năm họ làm việc ở GMT, trong khoảng từ 1 đến 10.
+  ![1 year GMT badge](https://assets.ppy.sh/profile-badges/GMT1y.png "1 year GMT badge") ![2 year GMT badge](https://assets.ppy.sh/profile-badges/GMT2y.png "2 year GMT badge") ![3 year GMT badge](https://assets.ppy.sh/profile-badges/GMT3y.png?2024 "3 year GMT badge") ![4 year GMT badge](https://assets.ppy.sh/profile-badges/GMT4y.png?2024 "4 year GMT badge") ![5 year GMT badge](https://assets.ppy.sh/profile-badges/GMT5y.png?2024 "5 year GMT badge") ![6 year GMT badge](https://assets.ppy.sh/profile-badges/GMT6y.png "6 year GMT badge") ![7 year GMT badge](https://assets.ppy.sh/profile-badges/GMT7y.png "7 year GMT badge") ![8 year GMT badge](https://assets.ppy.sh/profile-badges/GMT8y.png "8 year GMT badge") ![9 year GMT badge](https://assets.ppy.sh/profile-badges/GMT9y.png "9 year GMT badge") ![10 year GMT badge](https://assets.ppy.sh/profile-badges/GMT10y.png "10 year GMT badge")
+
+## Rời khỏi đội ngũ
+
+Khi một Kiểm duyệt viên quyết định rời khỏi đội ngũ, họ có thể nhận được những lợi ích sau đây tùy trường hợp:
+
+- **Danh hiệu [osu! Alumni](/wiki/People/osu!_Alumni)**: danh hiệu này được trao cho thành viên làm việc trong đội ngũ ít nhất 1 năm hoặc có đóng góp đáng kể cho nhóm trong thời gian ngắn hơn.
+- **osu!supporter**: Thành viên của GMT sẽ được trao một thẻ osu!supporter có thời hạn kéo dài gấp đôi thời gian làm việc trong nhóm, với cùng điều kiện của danh hiệu osu!Alumni ở trên.
+
+## Danh sách thành viên
+
+*Lưu ý: Tất cả các thành viên của GMT đều nói tiếng Anh bên cạnh những ngôn ngữ được liệt kê dưới đây trừ khi có ghi chú khác.*
+
+[Trang của Đội ngũ Kiểm duyệt viên](https://osu.ppy.sh/groups/4) liệt kê tất cả các thành viên trong đội ngũ. Bên cạnh những lĩnh vực được ghi dưới đây, tất cả các Kiểm duyệt viên đều tham gia vào [kiểm tra nội dung âm thanh và hình ảnh trong  beatmap](/wiki/Rules/Visual_content_considerations#getting-your-image-assessed).
+
+| Tên | Các ngôn ngữ khác | Mảng phụ trách chính |
+| :-- | :-- | :-- |
+
+### Sắp xếp theo ngôn ngữ
+
+| Kênh chat | Diễn đàn | Kiểm duyệt viên |
+| :-- | :-- | :-- |
+
+## Ghi chú
```

Comment:
the notes under this section do still need to be translated

Reply:
ci is not happy, pls send help :(

Reply:
it's telling you that the notes are unused because they would be used in the tables that you've omitted on purpose. you dont need to worry about that, it'll be fixed after generating the tables